symfony/stimulus-bundle/2.19/assets/controllers/csrf_protection_controller.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('submit', function (event) {
+    let csrfField = event.target.querySelector('input[data-controller="csrf-protection"]');
+
+    if (!csrfField) {
+        return;
+    }
+
+    let csrfCookie = csrfField.getAttribute('data-csrf-protection-cookie');
+
+    if (!csrfCookie) {
+        csrfField.setAttribute('data-csrf-protection-cookie', csrfCookie = csrfField.value);
+    }
+
+
+    if (!/^[-a-zA-Z0-9_]+$/.test(csrfCookie)) {
+        return
+    }
+
+    let csrfToken = btoa(String.fromCharCode.apply(null, (window.crypto || window.msCrypto).getRandomValues(new Uint8Array(18))));
```

Comment:
As this file will only be loaded when stimulus is used, and stimulus is [not compatible](https://stimulus.hotwired.dev/handbook/installing#browser-support) with IE11 (Turbo [neither](https://github.com/hotwired/hotwire-rails/issues/32#issuecomment-860091010))... 

.. i really think we can drop legacy code :) 


Question: if this recipe is released, is it possible to update it in the following weeks..  or will we need to release a new stimilus-bundle version ?

Reply:
Last time I checked, it worked on IE10. Since then, I moved this as a lazy stimulus controller.
It's all about the cost to support, and here, if the cost can be low (DX-wise), I'd rather favor a solution that includes as many users of the internet as possible.

Reply:
Ok i won't ask a thrid time 😅  thx